hw_7.py

```diff
@@ -0,0 +1,209 @@
+import json
+import os
+
+"""Задача-1
+У вас есть список(list) IP адрессов. Вам необходимо создать
+класс который будет иметь методы:
+1) Получить список IP адресов
+2) Получить список IP адресов в развернутом виде
+(10.11.12.13 -> 13.12.11.10)
+3) Получить список IP адресов без первых октетов
+(10.11.12.13 -> 11.12.13)
+4) Получить список последних октетов IP адресов
+(10.11.12.13 -> 13)
+"""
+
+
+class IPShow(object):
+    """Class to show different varieties of Ip list."""
+    def __init__(self):
+        self.__IP_list = ['135.146.235.135', '38.100.72.185', '127.27.201.98', '236.144.28.224', '57.80.195.145',
```

Comment:
this list should come from outside, here you just set this list from the argument like 
```
def __init__(self, ip_list):
     self._ip_list = ip_list
```